src/styles/themes/default.js

```diff
@@ -56,6 +56,7 @@ const darkTheme = {
     placeholderText: colors.greenIcons,
     heroCard: colors.blue,
     uploadPreviewActivityIndicator: colors.greenHighlightBackground,
+    dropUIBG: 'rgba(11,27,52,0.6)',
```

Comment:
```suggestion
    dropUIBG: addAplha(colors.dark, '99'),
```

Add a local function to the theme file.
```js
 // List of percentage to alphaHex mapping https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4
function addAlpha(hexColor, alphaHex){
  return `#${alphaHex}${hexColor.startsWith('#')? hexColor.slice(1) : hexColor}`;
}
```

Reply:
@shawnborton What color do you suggest to be used on the semi-transparent Drop UI from color.js?

Reply:
cc: @vladamx

Reply:
addAlpha does not work. 

I moved colors that i used to theme. 

I think you can make separate PR for rgba colors since i think that is out of scope for this PR.

Reply:
> out of scope for this PR.

Technically not because these variables are introduced in this PR. 


But it is fine for now.

Reply:
Yes, but there are still other colors that are also rgba.